tagstudio/src/cli/ts_cli.py

```diff
@@ -2625,14 +2627,14 @@ def scr_edit_entry_text(self, entry_index, field_index, allow_newlines=True, cle
 			self.lib.update_entry_field(
 				entry_index, field_index, new_content.rstrip('\n').rstrip('\r'), 'replace')
 
-	def scr_list_tags(self, query: str = '', tag_ids: list[int] = [], clear_scr=True) -> None:
+	def scr_list_tags(self, query: str = '', tag_ids: list[int] = None, clear_scr=True) -> None:
```

Comment:
if the empty list as a default parameter isnt intentional, then it might be causing some issues, see https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments